spec/features/user_show_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.feature 'User show page', type: :feature do
+  before do
+    @user = User.find_by(name: 'Tom')
+  end
+
+  scenario 'I can see the user\'s profile picture' do
+    visit user_path(@user)
+    expect(page).to have_css('.img-user-container img')
+  end
+
+  scenario 'I can see the user\'s username' do
+    visit user_path(@user)
+    expect(page).to have_content(@user.name)
+  end
+
+  scenario 'I can see the number of posts the user has written' do
+    visit user_path(@user)
+    expect(page).to have_content('Number of posts: 25')
+  end
+
+  scenario 'I can see the user\'s bio' do
+    visit user_path(@user)
+    expect(page).to have_css('.bio-container p', text: 'Brasilian student')
+  end
+
+  scenario 'When I click a user\'s post, it redirects me to that post\'s show page' do
+    visit user_path(@user)
+    first('.post-item a').click
+    expect(page.current_path).to include('/users/1/posts/1')
+  end
+
+  scenario 'When I click to see all posts, it redirects me to the user\'s post\'s index page' do
+    visit user_path(@user)
+    click_link 'See all posts'
+    expect(page).to have_current_path('/users/1/posts')
+  end
+end
```

Comment:
- Same issue is applicable here as the tests are failing. please, kindly fix them. Should they pass as contradicting this review comments, kindly add a screenshot of the test passed. 



![image](https://github.com/ulises2607/Blog-App/assets/59397159/7d81d299-9e74-42b8-8ce4-1988292010e5)